README.md

```diff
@@ -24,8 +24,13 @@ First set up a MongoDB server (locally or on [MongoDB Atlas](https://www.mongodb
 let mongooseURL = 'my_mongodb_url_with_key'
 export { mongooseURL as mongooseURL }
 ```
+On the server, you also need python3 and [TensorFlow Lite](https://www.tensorflow.org/lite/guide/python)  
```

Comment:
Replace with
`On the server, make sure you have Python3 and [TensorFlow Lite](https://www.tensorflow.org/lite/guide/python) installed.`